certificate-policy.md

```diff
@@ -1417,8 +1408,6 @@ Contents: This extension MUST contain at least one entry. Each entry MUST be eit
 
 Wildcard FQDNs are permitted.
 
-As of the Effective Date of these Requirements, prior to the issuance of a Certificate with a subjectAlternativeName extension or Subject commonName field containing a Reserved IP Address or Internal Name, the CA SHALL notify the Applicant that the use of such Certificates has been deprecated by the CA / Browser Forum and that the practice will be eliminated by October 2016. Also as of the Effective Date, the CA SHALL NOT issue a certificate with an Expiry Date later than 1 November 2015 with a subjectAlternativeName extension or Subject commonName field containing a Reserved IP Address or Internal Name. Effective 1 October 2016, CAs SHALL revoke all unexpired Certificates whose subjectAlternativeName extension or Subject commonName field contains a Reserved IP Address or Internal Name.
-
 #### 7.1.4.2.2. Subject Distinguished Name Fields
```

Comment:
If this CP is going to be restricted to issuing for US Government certificates we should be much more restrictive in the subject names forms - I suggest the following for 7.1.4.2.2:
a. Certificate Field: subject:commonName (OID 2.5.4.3) Required/Optional: Deprecated (Discouraged, but not prohibited) Contents: If present, this field MUST contain a single Fully-Qualified Domain Name that is one of the values contained in the Certificate's subjectAltName extension (see Section 7.1.4.2.1).
b. Certificate Field: subject:organizationName (OID 2.5.4.10) Required. Contents: The subject:organizationName field SHALL contain U.S. Government.
e. Certificate Field: subject:localityName (OID: 2.5.4.7) Optional Contents: If present, the subject:localityName field SHALL contain the Subject's locality information as verified under Section 3.2.2.1.
f. Certificate Field: subject:stateOrProvinceName (OID: 2.5.4.8) Optional  Contents: If present, the subject:stateOrProvinceName field SHALL contain the Subject's state information as verified under Section 3.2.2.1. 
g. Certificate Field: subject:postalCode (OID: 2.5.4.17) Optional Contents: If present, the subject:postalCode field MUST contain the Subject's zip or postal information as verified under Section 3.2.2.1.
h. Certificate Field: subject:countryName (OID: 2.5.4.6) Required. Contents: the subject:countryName SHALL contain the two-letter ISO 3166-1 country code US. 
i. Certificate Field: subject:organizationalUnitName Optional. The CA SHALL implement a process to verify this information in accordance with Section 3.2 
j. Other Subject Attributes All other optional attributes, when present within the subject field, SHALL contain information that has been verified by the CA. Optional attributes SHALL NOT contain metadata such as '.', '-', and ' ' (i.e. space) characters, and/or any other indication that the value is absent, incomplete, or not applicable.